### ISSUE TITLE
Store input names in the index

### ISSUE THREAD

**Issue by tmaklin** [algbio/themisto#44]
Can we add support for storing and printing the input file names (and their colors) to the next version that touches the index format? Fulgor for example has a `print-filenames` subcommand that we could then implement.

This would be helpful in checking the contents of previously built indexes and in mapping the pseudoalignment results back to the names of the sequences, which currently relies on having the input list file available.

**Comment by jnalanko** [comment]
Yes, definitely. Would it make sense to just write the names to a new file? I'm a bit hesitant about adding it into the  .tdbg or .tcolors files. A new file would also not break backward compatibility.

**Comment by tmaklin** [comment]
That should be fine since the syntax for themisto commands uses --index-prefix anyway to identify the index files.
